packages/host/app/components/operator-mode/stack-item.gts

```diff
@@ -53,9 +53,10 @@ interface Signature {
   };
 }
 
-export interface RenderedLinksToCard {
+export interface RenderedCardForOverlayActions {
```

Comment:
Since we now support a composite "contains" card to be opened on the stack, I renamed this to something more general and descriptive